### ISSUE TITLE
The low accuracy of CIFAR100

### ISSUE THREAD

**Issue by ziqi-zhang** [aaron-xichen/pytorch-playground#48]
Hi, thanks for sharing the code. I run the training scripts of CIFAR100 but only get an accuracy of 66% (without quantization), instead of 74%. Can you please provide the parameter setting of the training script?

Thanks,

**Comment by qilong-zhang** [comment]
@ziqi-zhang Hi, have you solved this issue? I also meet this problem.

**Comment by zhouT5** [comment]
yes ,I also meet this problem
